configure.ac

```diff
@@ -1,7 +1,7 @@
 dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.57)
-AC_INIT([libfabric], [0.0.2], [ofiwg@lists.openfabrics.org])
+AC_INIT([libfabric], [1.0.0-rc1], [ofiwg@lists.openfabrics.org])
```

Comment:
FWIW, I've seen a lot of packages not try to delimit "a" or "b" or "rc" from the version number.  I.e., "1.0.0rc1" (no . or -).  The version number parsers grab the various parts like this:

```perl
    # Typed by hand in github -- not checked!
    $version =~ m/(\d+)\.(\d+)\.(\d+)(.*)/;
    my $major = $1;
    my $minor = $2;
    my $release = $3;
    my $suffix = $4;
```